src/Shared/Data/ParticipantScreeningEpisodeRepository.cs

```diff
@@ -0,0 +1,27 @@
+
+using NHS.ServiceInsights.Model;
+
+namespace NHS.ServiceInsights.Data;
+
+public class ParticipantScreeningEpisodeRepository : IParticipantScreeningEpisodeRepository
+{
+    private readonly ServiceInsightsDbContext _dbContext;
+
+    public ParticipantScreeningEpisodeRepository(ServiceInsightsDbContext dbContext)
+    {
+        _dbContext = dbContext;
+    }
+
+    public bool CreateParticipantEpisode(ParticipantScreeningEpisode episode)
+    {
+        _dbContext.ParticipantScreeningEpisodes.Add(episode);
+
+        if (1 == _dbContext.SaveChanges())
+        {
+            return true;
+        }
+
+        return false;
```

Comment:
Did you try using the async methods for DbContext?

Reply:
made this change